lib/Bio/KBase/ObjectAPI/functions.pm

```diff
@@ -2882,9 +2886,9 @@ sub func_build_metagenome_metabolic_model {
 		max_objective_limit => $params->{max_objective_limit}
 	});
 	#Gapfilling model if requested
-	my $output = {};
+	my $function_output = {};
```

Comment:
`$output` had already been used earlier in this function. Please check that all the following renamed variables are correct.